README.md

```diff
@@ -25,15 +52,16 @@ This repo implements a pre-processing pipeline for the following documents. Curr
 	`pyenv  virtualenv 3.8.15 document-processing` <br />
 	`pyenv activate document-processing`
 
-# See the [Unstructured Quick Start](https://github.com/Unstructured-IO/unstructured#eight_pointed_black_star-quick-start) for the many OS dependencies that are required, if the ability to process all file types is desired.
+See the [Unstructured Quick Start](https://github.com/Unstructured-IO/unstructured#eight_pointed_black_star-quick-start) for the many OS dependencies that are required, if the ability to process all file types is desired.
+
 * Run `make install`
 * Start a local jupyter notebook server with `make run-jupyter` <br />
 	**OR** <br />
 	just start the fast-API locally with `make run-web-app`
 
-#### Extracting whatever from some type of document
+#### Testing the API
```

Comment:
This is really something like `Using the API` or the _local_ API, right? My first assumption was that this was going to be about running tests.